utils/cli/core/core.py

```diff
@@ -97,6 +98,16 @@ def tasks_create(self, name, labels, overlap, segment_size, bug, resource_type,
                 log.info(logger_string)
 
             self.tasks_upload(task_id, annotation_format, annotation_path, **kwargs)
+        if dataset_repository_url:
+            git_url = self.api.git
```

Comment:
@dreaquil When trying to create a task, I got the following error
```python
Traceback (most recent call last):
  File "/home/andrey/workspace/cvat/utils/cli/cli.py", line 42, in <module>
    main()
  File "/home/andrey/workspace/cvat/utils/cli/cli.py", line 34, in main
    actions[args.action](cli, **args.__dict__)
  File "/home/andrey/workspace/cvat/utils/cli/core/core.py", line 102, in tasks_create
    git_url = self.api.git
AttributeError: 'CVAT_API_V1' object has no attribute 'git'
```

Reply:
Apologies.. sems to not have pushed. Pushed now.